tests/unittests/Foundation/NSOperationTests.mm

```diff
@@ -59,3 +62,102 @@
     ASSERT_FALSE([cancelledOperation isExecuting]);
     ASSERT_TRUE([cancelledOperation isCancelled]);
 }
+
+// Test subclass for NSOperation
+@interface MyOperation : NSOperation
+
+@property (assign, getter = isExecuting) BOOL executing;
+@property (assign, getter = isFinished, readonly) BOOL finished;
+
+@end
+
+@implementation MyOperation
+
+@synthesize executing = _executing;
+@synthesize finished = _finished;
+
+- (void)setExecuting:(BOOL)executing
+{
+    [self willChangeValueForKey:@"isExecuting"];
+    [self willChangeValueForKey:@"isFinished"];
```

Comment:
I don't think it's necessary to test the test class MyOperation, but I'll add KVO tests for the base class.

---
In reply to: [69353346](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69353346) [](ancestors = 69353346)